docs/reference/encryption-and-security.md

```diff
@@ -0,0 +1,154 @@
+# Encryption and Security
+
+## Overview
```

Comment:
Out of the box default is a simple AES Key, with manual rotation (KEY + KEY ID, and you can rotate out by setting DEPRECATED_KEY_ID https://github.com/friggframework/frigg/blob/67b1ea454e7d25e4d58ff2ba08b025ca4141aa26/packages/core/encrypt/Cryptor.js#L53C1-L54C1)

Worth documenting that here, and saying that we've added support for KMS for those who want a more robust/managed solution.